pypeit/core/extract.py

```diff
@@ -1143,7 +1144,7 @@ def objfind(image, thismask, slit_left, slit_righ, inmask=None, fwhm=3.0, use_us
             boxcar_rad_skymask=None, cont_sig_thresh=2.0,
             skymask_nthresh=1.0, specobj_dict=None, cont_fit=True, npoly_cont=1, find_min_max=None,
             show_peaks=False, show_fits=False, show_trace=False, show_cont=False, debug_all=False,
-            qa_title='objfind'):
+            qa_title='objfind', objfindQA_dict=None):
```

Comment:
See my comment below. You should only be passing in the QA outfile here, not this dictionary.